scripts/main/photo.js

```diff
@@ -843,11 +848,12 @@ photo.editTags = function (photoIDs) {
 /**
  * @param {string[]} photoIDs
  * @param {string[]} tags
+ * @param {boolean} override
```

Comment:
Same thing here. I would prefer something like
```suggestion
 * @param {boolean} shallOverride
```